UnityProject/Assets/Scripts/Core/Sprite Handler/SpriteHandler.cs

```diff
@@ -887,15 +887,15 @@ public void UpdateMe()
 
 		if (isAnimation == false)
 		{
-			UpdateManager.Remove(CallbackType.UPDATE, UpdateMe);
+			UpdateManager.Remove(CallbackType.LATE_UPDATE, UpdateMe);
 		}
 	}
 
 	private void InternalStopAnimation()
 	{
 		InternalChangeSprite(CataloguePage + 1 < SubCatalogue.Count ? CataloguePage + 1 : 0, false);
 		isAnimation = false;
-		UpdateManager.Remove(CallbackType.UPDATE, UpdateMe);
+		UpdateManager.Remove(CallbackType.LATE_UPDATE, UpdateMe);
```

Comment:
why late update?

Reply:
it moves the sprites into a separate update list that optimise the main update list, Should I move back it doesn't really matter Either way